### PR TITLE
[misc] Better README page showing contributions

### PR DESCRIPTION
**Pull request by archibate**
Related issue = https://github.com/taichi-dev/taichi/pull/1049#issuecomment-633392852

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

Moving discussions here!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
...
+
+The Simplified Chinese documentation (简体中文文档) was created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
+  - [Rex (rexwangcc)](https://github.com/rexwangcc)
+  - [jianzhouwan (ArkhamWJZ)](https://github.com/ArkhamWJZ)
+
+[[List of all contributors to the Simplified Chinese documentation of Taichi]](https://github.com/taichi-dev/taichi-docs-zh-cn/graphs/contributors)
+
+-------------------------------
```
Comment (archibate) [inline README.md:72]: I think we should move this part to taichi-docs-zh-cn.

File: README.md [hunk 2]
```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
...
+
+[[List of all contributors to Taichi]](https://github.com/taichi-dev/taichi/graphs/contributors)
+
+-------------------------------
+
+The Simplified Chinese documentation (简体中文文档) was created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
+  - [Rex (rexwangcc)](https://github.com/rexwangcc)
+  - [jianzhouwan (ArkhamWJZ)](https://github.com/ArkhamWJZ)
```
Comment (ArkhamWJZ) [inline README.md:68]: Btw, may I use my Chinese name here? (*/ω＼*)
```suggestion
  - [万健洲 (ArkhamWJZ)](https://github.com/ArkhamWJZ)
```

**Comment by yuanming-hu** [comment]
I think we can do both: keep a precise version in the README of `taichi` and a more detailed version in the Chinese documentation README.

**Comment by archibate** [comment]
-1, I don't think English Taichi users will be interested in zh-cn doc contributors anyway.
Just like we don't put taichi_element contributors in taichi README regardless their great work.

**Comment by yuanming-hu** [comment]
> -1, I don't think English Taichi users will be interested in zh-cn doc contributors anyway.
> Just like we don't put taichi_element contributors in taichi README regardless their great work.

It's not about users' interest. It's about acknowledgment. Most people don't care about the closing credits after a movie ends, but the credits are still there.

**Comment by archibate** [comment]
Maybe we should ask @KLozes for the feeling about putting taichi-docs-zh-cn contributors to taichi main repo README as an en-us user? I don't like this anyway.

**Comment by yuanming-hu** [comment]
Thanks for all the discussions here. I think @archibate's opinion makes sense to some extent, but after taking all factors into consideration, I'd stick to the current format of the contributor list.

**Pull request closed without merging**